src/RegExLib.Core/Entities/Author.cs

```diff
@@ -29,9 +31,8 @@ public void AddExpression(Expression expression)
             _expressions.Add(expression);
         }
 
-        public override string ToString()
+        public override string? ToString()
         {
-            Guard.Against.NullOrWhiteSpace(FullName, nameof(FullName));
             return FullName;
         }
```

Comment:
I would remove the `?` on the return type.

Is `FullName` truly nullable? If so, I'd default the return value to an empty string or other fallback value as a courtesy to the calling code, so it doesn't have to perform a null check:

```csharp
public override string ToString() => Fullname ?? "";
```

If not, this should do:

```csharp
public override string ToString() => Fullname;
```

Reply:
@antmdvs Done